pkg/reconciler/pipeline/controller.go

```diff
@@ -725,6 +726,15 @@ func (r *pipelineReconciler) updateDesiredState(ctx context.Context, pl *dfv1.Pi
 }
 
 func (r *pipelineReconciler) resumePipeline(ctx context.Context, pl *dfv1.Pipeline) (bool, error) {
+
+	// reset pause timestamp
+	if pl.GetAnnotations()[dfv1.KeyPauseTimestamp] != "" {
+		err := r.client.Patch(ctx, pl, client.RawPatch(types.JSONPatchType, []byte(dfv1.RemovePauseTimestampPatch)))
+		if err != nil && !apierrors.IsNotFound(err) {
+			return true, err
+		}
```

Comment:
Just a tiny issue here:
if err != nill && IsNotFound, return false, nil

Somehow the pipeline can not be found, and we skip it.

Reply:
Updated to `if err != nil && apierrors.IsNotFound(err) {
			return false, nil
		}`

Reply:
```
if err != nil {
  if apierrors.IsNotFound(err) {
    return false, nil // skip pl if we can't find it
  } else {
      return false, err // otherwise requeue resume
  }
}
```